### DISCUSSION TITLE
Rotate Color Picker hue bar

### DISCUSSION THREAD

**Discussion by jonaaldas** [primefaces/community#2635] [PrimeVue] [answered]
Hi all I want to rotate the huebar to scroll from left to right instead of top to bottom like the default behaviour. If anybody knows how to do it please let me know. 
Thanks 
<img width="456" alt="image" src="https://github.com/user-attachments/assets/27e3cc80-7a6e-4170-a37e-48b2e24dae39">
<img width="247" alt="image" src="https://github.com/user-attachments/assets/45c074a0-ab63-437b-9ffd-62c6ec7cde2f">

**Comment by qwadrox** [comment, marked as answer]
Hi @jonaaldas 

I don't think you can do that because the component internally calculates drag only on the Y axis, and it's assuming that 150 is the height of the hue slider.
https://github.com/primefaces/primevue/blob/35646338891494f055fb46928a83fa3aba6a368d/packages/primevue/src/colorpicker/ColorPicker.vue#L105-L111

But it could be a good idea to add vertical slider in a next feature.

**Reply by qwadrox** [reply]
@tugcekucukoglu
Can you tag it maybe as feature request ?

**Reply by jonaaldas** [reply]
Hi @qwadrox thank you for your response. 
Bummer hopefully they can add it in a future release.
